_posts/2016-11-09-Rust-1.13.md

```diff
@@ -0,0 +1,475 @@
+[rustc-benchmarks]: https://github.com/rust-lang-nursery/rustc-benchmarks
+[inline]: https://github.com/rust-lang/rust/pull/36524
+
+#### Library stabilizations
+
+* [Add `assert_ne!` and `debug_assert_ne!`][35074]
+* [Make `vec_deque::Drain`, `hash_map::Drain`, and `hash_set::Drain`
+  covariant][35354]
+* [Implement `AsRef<[T]>` for `std::slice::Iter`][35559]
+* [Implement `Debug` for `std::vec::IntoIter`][35707]
+* [`CString`: avoid excessive growth just to 0-terminate][35871]
+* [Implement `CoerceUnsized` for `{Cell, RefCell, UnsafeCell}`][35627]
+* [Use arc4rand on FreeBSD][35884]
+* [memrchr: Correct aligned offset computation][35969]
+* [Improve Demangling of Rust Symbols][36059]
+* [Use monotonic time in condition variables][35048]
+* [Implement `Debug` for `std::path::{Components,Iter}`][36101]
+* [Implement conversion traits for `char`][35755]
+* [Fix illegal instruction caused by overflow in channel cloning][36104]
+* [Zero first byte of CString on drop][36264]
+* [Inherit overflow checks for sum and product][36372]
+* [Add missing Eq implementations][36423]
+* [Implement `Debug` for `DirEntry`][36631]
+* [When `getaddrinfo` returns `EAI_SYSTEM` retrieve actual error from
+  `errno`][36754]
+* [`SipHasher`] is deprecated. Use [`DefaultHasher`].
+* [Implement more traits for `std::io::ErrorKind`][35911]
+* [Optimize BinaryHeap bounds checking][36072]
+* [Work around pointer aliasing issue in `Vec::extend_from_slice`,
+  `extend_with_element`][36355]
+* [Fix overflow checking in unsigned pow()][34942]
+
+[35074]: https://github.com/rust-lang/rust/pull/35074
+[35074]: https://github.com/rust-lang/rust/pull/35074
+[35354]: https://github.com/rust-lang/rust/pull/35354
+[35559]: https://github.com/rust-lang/rust/pull/35559
+[35707]: https://github.com/rust-lang/rust/pull/35707
+[35871]: https://github.com/rust-lang/rust/pull/35871
+[35627]: https://github.com/rust-lang/rust/pull/35627
+[35884]: https://github.com/rust-lang/rust/pull/35884
+[35969]: https://github.com/rust-lang/rust/pull/35969
+[36059]: https://github.com/rust-lang/rust/pull/36059
+[35048]: https://github.com/rust-lang/rust/pull/35048
+[36101]: https://github.com/rust-lang/rust/pull/36101
+[35755]: https://github.com/rust-lang/rust/pull/35755
+[36104]: https://github.com/rust-lang/rust/pull/36104
+[36264]: https://github.com/rust-lang/rust/pull/36264
+[36372]: https://github.com/rust-lang/rust/pull/36372
+[36423]: https://github.com/rust-lang/rust/pull/36423
+[36631]: https://github.com/rust-lang/rust/pull/36631
+[36754]: https://github.com/rust-lang/rust/pull/36754
+[35911]: https://github.com/rust-lang/rust/pull/35911
+[36072]: https://github.com/rust-lang/rust/pull/36072
+[36355]: https://github.com/rust-lang/rust/pull/36355
+[34942]: https://github.com/rust-lang/rust/pull/34942
+
+See the [detailed release notes][notes] for more.
+
+#### Cargo features
+
+This release contains critical security updates to Cargo, which depends on
```

Comment:
I'm a bit worried that this critical security update isn't being flagged high enough in the post -- feels a bit like it's buried? Maybe just forward-reference it?